Libs/DICOM/Widgets/ctkDICOMTableView.cpp

```diff
@@ -615,6 +675,92 @@ void ctkDICOMTableView::addSqlWhereCondition(const QString column, const QString
   d->sqlWhereConditions.insert(column, values);
 }
 
+//------------------------------------------------------------------------------
+void ctkDICOMTableView::removeSqlWhereCondition(const QString column)
+{
+  Q_D(ctkDICOMTableView);
+  d->sqlWhereConditions.remove(column);
```

Comment:
All the `removeSql[...]` method could be updated to return a bool on success ...
```
  return d->sqlWhereConditions.remove(column) > 0;
```